### PR TITLE
Minor cleanup of depreactions

### PR DESCRIPTION
**Pull request by StefMa**
As discussed in #2 `valueUnsafe` shouldn't be used. 
There are better functions for them.

Beside of that I removed an 1 year old deprecated function. I think this is okay 😇

### CODE REVIEW

File: lib/src/main/kotlin/se/gustavkarlsson/koptional/Optional.kt [hunk 1]
```diff
@@ -4,6 +4,7 @@ sealed class Optional<out T : Any> {
 
     abstract val value: T?
 
+    @Deprecated(message = "Unsafe to use. Use `ifPresent` or `ifAbsent` instead")
```
Comment (gustavkarlsson) [inline lib/src/main/kotlin/se/gustavkarlsson/koptional/Optional.kt]: `ifPresent` and `ifAbsent` are not really alternatives to `valueUnsafe` . They are rather alternatives to use cases that might use `valueUnsafe`.

I would change it to just "Useless" and add `ReplaceWith("value!!")` or something like that.

**Review by gustavkarlsson — CHANGES_REQUESTED** [review]
Thx for the contribution!

**Review by gustavkarlsson — APPROVED** [review]

File: lib/src/main/kotlin/se/gustavkarlsson/koptional/Optional.kt [hunk 2]
```diff
@@ -4,6 +4,10 @@ sealed class Optional<out T : Any> {
 
     abstract val value: T?
 
+    @Deprecated(
+        message = "Useless"
+        replaceWith = ReplaceWith("value!!")
```
Comment (gustavkarlsson) [inline lib/src/main/kotlin/se/gustavkarlsson/koptional/Optional.kt]:
```suggestion
        message = "Useless",
        replaceWith = ReplaceWith("value!!")
```

**Review by gustavkarlsson — CHANGES_REQUESTED** [review]
Missing comma ;)

**Comment by StefMa** [comment]
🤦 Thanks

**Pull request merged**